docs/sources/administration/plugin-management/index.md

```diff
@@ -33,19 +37,19 @@ Use panel plugins when you want to:
 
 ## Data source plugins
 
-Data source plugins add support for new databases, such as [Google BigQuery](/grafana/plugins/grafana-bigquery-datasource).
+Data source plugins add support for new databases, such as [Google BigQuery](/grafana/plugins/grafana-bigquery-datasource), [Zendesk](https://grafana.com/grafana/plugins/gabrielthomasjacobs-zendesk-datasource/) and [NetData](https://grafana.com/grafana/plugins/netdatacloud-netdata-datasource/) Data source plugins.
 
 Data source plugins communicate with external sources of data and return the data in a format that Grafana understands. By adding a data source plugin, you can immediately use the data in any of your existing dashboards.
 
-Use data source plugins when you want to import data from external systems.
+Use data source plugins when you want to import data from external or 3rd party systems.
```

Comment:
This is a slightly strange sentence given data source plugins are also how we query things like loki whether thats in the same Grafana stack in cloud or self hosted.

Reply:
I actually took it from https://grafana.com/developers/plugin-tools/introduction/plugin-types-usage#usage-of-data-source-plugins where it says "when you want to work with data provided by a third-party service".